packages/core/src/lib/database/migrations/1749490131969-CreateTableCamshots.ts

```
@@ -0,0 +1,391 @@
+import { DatabaseTypeEnum } from '@gauzy/config';
+
+export class CreateTableCamshots1749490131969 implements MigrationInterface {
+	name = 'CreateTableCamshots1749490131969';
+
+	/**
+	 * Up Migration
+	 *
+	 * @param queryRunner
+	 */
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		console.log(chalk.yellow(this.name + ' start running!'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)